api/users.go

```diff
@@ -36,10 +39,11 @@ func GetUsers(c *gin.Context) {
 func GetUser(c *gin.Context) {
 	user, err := store.GetUserLogin(c, c.Param("login"))
 	if err != nil {
-		c.String(404, "Cannot find user. %s", err)
-	} else {
-		c.JSON(200, user)
+		c.AbortWithError(http.StatusNotFound, fmt.Errorf("Cannot find user. %s", err))
```

Comment:
I personally expect a json response from a json api and not a text reponse, even when it's an error :)